files/scripts/write_standby.py

```diff
@@ -68,9 +77,18 @@ def asic_db(self):
         Returns the ASIC DB connector.
         Initializes the connector during the first call
         """
-        if self.asic_db_connector is None:
-            self.asic_db_connector = SonicV2Connector()
-            self.asic_db_connector.connect('ASIC_DB')
+        retries = 3
+        while self.asic_db_connector is None and retries > 0:
+            try:
+                self.asic_db_connector = SonicV2Connector()
```

Comment:
Same as above, would `use_unix_socket_path=True` fix this?

Reply:
Changed.